src/frontend/components/left-navbar/Left-navbar.js

```diff
@@ -0,0 +1,78 @@
+import "./left-navbar.css";
+import { FaFeatherAlt } from "react-icons/fa";
+import { AiFillHome } from "react-icons/ai";
+import { RiHashtag } from "react-icons/ri";
+import { FiBookmark } from "react-icons/fi";
+import { CgProfile } from "react-icons/cg";
+import { Link } from "react-router-dom";
+import { PostModal } from "../index";
+import { useDispatch, useSelector } from "react-redux";
+import { addPost } from "../../features/post-slice";
+
+export function LeftNav({ postModal, setPostModal }) {
+  const token = useSelector((store) => store.auth.token);
```

Comment:
Suggestion: It's my opinion it's good to use optional chaining (?) to check that each reference in the chain is valid, you can differ